src/components/rangeFilterInputWithSlider/index.tsx

```diff
@@ -24,14 +27,17 @@ type ChangeRangeInputWithSliderCallback<Name> = {
   changeType?: 'inputfield' | 'slider';
 } & ChangeCallback<Name>;
 
-type RangeSliderProps = {
-  facets?: Facet[];
-  rangeSliderScale?: number[];
-  chartHeight?: RangeSliderWithChartProps['chartHeight'];
-} & (
-  | { facets: Facet[]; chartHeight?: string; rangeSliderScale?: never }
-  | { rangeSliderScale: number[]; facets?: never; chartHeight?: never }
-);
+type RangeSliderProps =
+  | {
+      facets: Facet[];
+      chartHeight?: RangeSliderWithChartProps['chartHeight'];
+      rangeSliderScale?: undefined;
```

Comment:
what's the background for changing this type? The previous state was more restrictive which made sense in my opinion it made it clear that when facets exists, we are in "chart mode"